packages/integration-sdk-benchmark/src/index.js

```diff
@@ -1 +1,3 @@
 require('./benchmarks/FileSystemGraphObjectStore.addEntities');
+require('./benchmarks/InMemoryGraphObjectStore.addEntities');
+require('./benchmarks/JobState.addEntities');
```

Comment:
I'm not sure if removing the `index.js` file is the right thing to do, but it isn't a good way to run all the benchmarks.

Reply:
I'd support the removal. A different approach seems to be needed.